tests/signature.spec.coffee

```diff
@@ -634,6 +634,31 @@ describe 'Signature:', ->
 						bar: 'hello world'
 					done()
 
+		describe 'given a multi-word signature with a variadic required parameter', ->
+
+			beforeEach ->
+				@signature = new Signature('foo bar <baz...>')
+
+			it 'should throw an error if prefix is different', (done) ->
+				@signature.compileParameters 'foo baz hello world', (error, result) ->
+					expect(error).to.be.an.instanceof(Error)
+					expect(result).to.not.exist
+					done()
+
+			it 'should all parameters together if command exceeds', (done) ->
```

Comment:
Was ths supposed to be something like "should **group** all parameters together if command **succeeds**" ?